alvr/server/src/hand_gestures.rs

```diff
@@ -0,0 +1,601 @@
+use std::{
+    collections::HashMap,
+    hash::Hash,
+    time::{Duration, SystemTime, UNIX_EPOCH},
+};
+
+use alvr_common::{
+    glam::{Vec2, Vec3},
+    once_cell::sync::Lazy,
+    *,
+};
+
+use alvr_packets::ButtonValue;
+use alvr_session::HandGestureConfig;
+
+use crate::input_mapping::ButtonMappingManager;
+
+fn lerp_pose(a: Pose, b: Pose, fac: f32) -> Pose {
+    Pose {
+        orientation: a.orientation.lerp(b.orientation, fac),
+        position: a.position.lerp(b.position, fac),
+    }
+}
+
+pub static HAND_GESTURE_PROFILE_INFO: Lazy<InteractionProfileInfo> =
```

Comment:
You don't need to construct a `InteractionProfileInfo`. you are giving it a `QUEST_CONTROLLER_PROFILE_PATH` which is wrong. just store the button set HashSet.